test/ValidationFramework/MissionSequence/MissionSequence.hpp

```diff
@@ -0,0 +1,153 @@
+using ostk::math::geometry::d3::objects::Point;
+using ostk::math::object::Matrix3d;
+using ostk::math::object::Vector3d;
+using ostk::math::object::VectorXd;
+
+using ostk::physics::data::Scalar;
+using ostk::physics::Environment;
+using ostk::physics::coord::Frame;
+using ostk::physics::coord::Position;
+using ostk::physics::coord::Velocity;
+using ostk::physics::environment::Object;
+using ostk::physics::environment::object::Celestial;
+using ostk::physics::environment::object::celestial::Earth;
+using ostk::physics::environment::object::celestial::Moon;
+using ostk::physics::environment::object::celestial::Sun;
+using ostk::physics::environment::ephemerides::Analytical;
+using ostk::physics::time::DateTime;
+using ostk::physics::time::Duration;
+using ostk::physics::time::Instant;
+using ostk::physics::time::Interval;
+using ostk::physics::time::Time;
+using ostk::physics::time::Scale;
+using ostk::physics::units::Angle;
+using ostk::physics::units::Derived;
+using ostk::physics::units::Length;
+using ostk::physics::units::Mass;
+using ostk::physics::environment::object::celestial::Earth;
+using EarthGravitationalModel = ostk::physics::environment::gravitational::Earth;
+using EarthMagneticModel = ostk::physics::environment::magnetic::Earth;
+using EarthAtmosphericModel = ostk::physics::environment::atmospheric::Earth;
+using SWManager = ostk::physics::environment::atmospheric::earth::Manager;
+using ostk::physics::environment::atmospheric::earth::CSSISpaceWeather;
+
+using ostk::astro::eventcondition::InstantCondition;
+using ostk::astro::trajectory::State;
+using ostk::astro::trajectory::Sequence;
+using ostk::astro::trajectory::Propagator;
+using ostk::astro::trajectory::Orbit;
+using ostk::astro::trajectory::LocalOrbitalFrameFactory;
+using ostk::astro::trajectory::LocalOrbitalFrameDirection;
+using ostk::astro::trajectory::state::NumericalSolver;
+using ostk::astro::trajectory::state::CoordinatesSubset;
+using ostk::astro::trajectory::state::CoordinatesBroker;
+using ostk::astro::trajectory::state::coordinatessubsets::CartesianPosition;
+using ostk::astro::trajectory::state::coordinatessubsets::CartesianVelocity;
+using ostk::astro::trajectory::state::NumericalSolver;
+using ostk::astro::trajectory::orbit::models::kepler::COE;
+using ostk::astro::Dynamics;
+using ostk::astro::flight::system::PropulsionSystem;
+using ostk::astro::flight::system::SatelliteSystem;
+using ostk::astro::dynamics::PositionDerivative;
+using ostk::astro::dynamics::CentralBodyGravity;
+using ostk::astro::dynamics::ThirdBodyGravity;
+using ostk::astro::dynamics::AtmosphericDrag;
+using ostk::astro::dynamics::Thruster;
+using ostk::astro::guidancelaw::ConstantThrust;
+using ostk::astro::guidancelaw::QLaw;
+using ostk::astro::solvers::FiniteDifferenceSolver;
+
+static const Derived::Unit GravitationalParameterSIUnit =
+    Derived::Unit::GravitationalParameter(Length::Unit::Meter, ostk::physics::units::Time::Unit::Second);
+
+class MissionSequence
```

Comment:
Might want to add all the file output writing and common func into an `Application` abstract class in the future (probably when we start adding a second `App`), what do you think?

Reply:
> (probably when we start adding a second App)

Yeah that was my point with this, we can refactor when we have more later maybe. Let's see